### DISCUSSION TITLE
Questions about PhoneProfilesPlusExtender

### DISCUSSION THREAD

**Discussion by henrichg** [henrichg/PhoneProfilesPlusExtender#5] [Q&A]
Write me questions about PPPE. I will answer it.

**Comment by Richard-Br** [comment]
Hallo Henrich, just upgraded a Pixel 7 Pro to Android 14 and now the Extender can no longer be activated?  In the Android Accessibility settings, under Downloaded apps, it is greyed out with text "Off" showing.
Press on it and Android says it is a restricted setting, not available at this moment, because of security. 

This obviously disables specific PPP settings (red warnings).

What to do?

**Comment by henrichg** [comment]
Extender displays information about "Restricted settings". Open Extender and read it.

"If is not possible to enable Accessibility service, maybe are not allowed restricted settings for PhoneProfilesPlusExtender.
Click "App info" button to allow it.
In "App info" screen, click menu at top and in it "Allow restricted settings".
"Allow restricted settings" is not displayed, when restricted settings are allowed.
However, if PhoneProfilesPlusExtender is installed from a downloaded apk file, "Allow restricted settings" may not appear at all.
Then you must uninstall it, and install it from supported store, for example Droid-ify."

"App info" screen is screen in device Settings/Apps/PhoneProfilesPlusExtender.

**Reply by Richard-Br** [reply]
Yes! Thank you for the quick reply. Looks like security by obscurity...
Thanks for making a really useful app!

**Comment by henrichg** [comment]
Restricted setting is also "Notification access" used in PPP "Notification sensor". This is new security "feature", added in Android 14.

**Reply by katastrophe92** [reply]
Hi hi! Does it work on android 14 or not? Thanks

**Reply by henrichg** [reply]
Yes, working also in Android 14. You must enable "Restricted settings" also in Android 14.

But, as for Extender, PPP must be installed from store, recommended is Droid-ify.
